torch/distributions/relaxed_bernoulli.py

```diff
@@ -47,6 +48,11 @@ def __init__(self, temperature, probs=None, logits=None, validate_args=None):
             batch_shape = self._param.size()
         super(LogitRelaxedBernoulli, self).__init__(batch_shape, validate_args=validate_args)
 
+    @property
+    def _reshape_args(self):
+        yield 'temperature', False
```

Comment:
`temperature` doesn't appear to be included in `broadcast_all` in the constructor, so I've omitted it from reshaping here.